bundles/org.openhab.binding.evcc/src/main/resources/OH-INF/thing/loadpoint.xml

```diff
@@ -0,0 +1,614 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="evcc"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<thing-type id="loadpoint" listed="false">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"/>
+		</supported-bridge-type-refs>
+		<label>Loadpoint</label>
+		<description>A loadpoint configured in your evcc instance</description>
+	</thing-type>
+	<channel-type id="loadpointBatteryBoost">
+		<item-type>Switch</item-type>
+		<label>Battery Boost</label>
+		<description>Controls and indicates whether battery boost is active or not</description>
+		<category>Switch</category>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Control</tag>
+		</tags>
+		<state>
+			<options>
+				<option value="ON">Activated</option>
+				<option value="OFF">Deactivated</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="loadpointChargeDuration">
+		<item-type>Number:Time</item-type>
+		<label>Charging Duration</label>
+		<description>Time passed since start of the charging</description>
+		<category>Time</category>
+		<state pattern="%.1f min" readOnly="true"/>
+	</channel-type>
+	<channel-type id="loadpointChargePower">
+		<item-type>Number:Power</item-type>
+		<label>Charging Power</label>
+		<description>Current power of charging</description>
+		<category>Energy</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Power</tag>
+		</tags>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="loadpointChargeTotalImport">
+		<item-type>Number:Energy</item-type>
+		<label>Charge Total Import</label>
+		<description>Total imported energy</description>
+		<category>Energy</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Energy</tag>
+		</tags>
+		<state pattern="%.1f %unit%" readOnly="true"></state>
+	</channel-type>
+	<channel-type id="loadpointChargeRemainingDuration">
+		<item-type>Number:Time</item-type>
+		<label>Charging Remaining Duration</label>
+		<description>Remaining duration until limit SoC is reached</description>
+		<category>Time</category>
+		<state pattern="%.1f min" readOnly="true"/>
```

Comment:
Could you aded semantic tags? Not sure if this is Calculation (exact) or Forecast (estimate) 
Forecast+Duration